alcs-api/docker-compose.yml

```diff
@@ -26,6 +27,7 @@ services:
       - postgres:/var/lib/postgres
     networks: 
       - alcs
+
   keycloak:
```

Comment:
since we switched to government sso do we need this local keycloak?

Reply:
Nope, I am happy to remove it

Reply:
Done! Also removed admin-db